crates/forge/tests/data/steps/src/lib.cairo

```diff
@@ -0,0 +1,60 @@
+#[cfg(test)]
+mod tests {
+    #[test]
+    // require 570031 steps
+    fn steps_570031() {
+        let mut i = 0;
+
+        loop {
+            if i == 30_000 {
+                break;
+            }
+
+            i = i + 1;
```

Comment:
[nit] We could probably perform some more complicated operations to reach the expected steps faster